app/policies/delivery_partner_profile_policy.rb

```diff
@@ -1,17 +1,19 @@
 # frozen_string_literal: true
 
 class DeliveryPartnerProfilePolicy < ApplicationPolicy
-  def create?
```

Comment:
Weren't those used? could very well be that they weren't, Might be worth asking in the dev channel

Reply:
@StupidCodeFactory it looks like there was a previous model `DeliveryPartnerProfile` which had migration and everything. Migration and model was removed, but other stuff leftover, like this policy.